pkg/kubelet/volume/empty_dir/empty_dir.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Packages should usually be no underscore, all one mashed together thing (httptest, filepath, etc).  Minor consistency thing, but...